README.md

```diff
@@ -1 +1,59 @@
-# JavaScript-Capstone
\ No newline at end of file
+# JavaScript-Capstone
+
+> The app lists different foods from different areas in the word with opions from people worldwide.
+
+
+## Built With
+
+- JavaScript, HTML, CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://harlexkhal.github.io/Foodipy/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+
+
+### Setup
+
+1. git clone https://github.com/harlexkhal/Foodipy.git
+2. cd Foodipy
+3. npm install
+4. npm start to serve into local 8080 port
+
+### Install
+
+1. git clone https://github.com/harlexkhal/Foodipy.git
+2. cd Foodipy
+3. npm install
+4. npm start to serve into local 8080 port
+
+### Deployment
+https://harlexkhal.github.io/Foodipy/
+
+## Authors
+
+👤 **Mnemba Chambuya**
+
+- GitHub: [@mnekx](https://github.com/mnekx)
+- Twitter: [@MnembaChambuya](https://twitter.com/MnembaChambuya)
+- LinkedIn: [mnemba-chambuya](https://linkedin.com/in/mnemba-chambuya)
+
+## Screenshot ![see](./src/assets/images/screenshot.png?raw=true "Title")
```

Comment:
That is an interesting point of view and I agree with you. It could provide a better overview of the app. But, I used the [Microverse README.md template](https://github.com/microverseinc/readme-template). Could we make the improvement?

Reply:
Yeah, sure we can 🚀

Reply:
This is an example of a readme I've done on a previous project https://github.com/harlexkhal/Leaderboard/blob/development/README.md

Reply:
Done